esrally/driver/runner.py

```diff
@@ -23,6 +23,8 @@
 import types
 from collections import Counter, OrderedDict
 from copy import deepcopy
+from os.path import commonprefix
```

Comment:
While I see that it's useful, I find it a bit odd to use a path-related library method for string processing. Although it's not ideal I'm ok if we keep it here.